src/settings-ui/Settings.UI/SettingsXAML/Views/GeneralPage.xaml.cs

```diff
@@ -116,7 +116,14 @@ public static int UpdateUIThemeMethod(string themeName)
 
         private void OpenColorsSettings_Click(object sender, RoutedEventArgs e)
         {
-            Helpers.StartProcessHelper.Start(Helpers.StartProcessHelper.ColorsSettings);
+            try
+            {
+                Helpers.StartProcessHelper.Start(Helpers.StartProcessHelper.ColorsSettings);
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError($"Error while trying to open the system color settings: {ex.Message}");
```

Comment:
```suggestion
                Logger.LogError("Error while trying to open the system color settings", ex);
```

Does it make sense to log the stacktrace?